visual_behavior/visualization/utils.py

```diff
@@ -6,16 +6,16 @@
 
 
 def get_container_plots_dir():
-    return r'//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/qc_plots/container_plots'
+    return r'//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/qc_plots/container_plots_sdk_test'
```

Comment:
Do you want these path changes to be merged into dev, and subseqently into master? I'd assume not.

Reply:
ah good catch, thanks! ill fix it